### PR TITLE
BACKLOG-14531 : add graphQL playground into developer tools

### PR DESCRIPTION
**Pull request by Yousria**

### CODE REVIEW

File: graphql-dxm-provider/src/main/resources/tools/graphql-playground.jsp [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <title>GraphQL Playground</title>
+        <link rel="stylesheet" href="${modulePath}/css/graphql-playground.css" type="text/css"/>
+    </head>
+    <body>
+        <div id="graphql-playground">
+            <p class="loading">Loading <span class="title">GraphQL Playground</span></p>
+        </div>
+        <script src="${modulePath}/javascript/graphql-playground-1.7.23.js"></script>
+        <script>
+            const customTheme = {
```
Comment (cedricmoitrier) [inline graphql-dxm-provider/src/main/resources/tools/graphql-playground.jsp:20]: You can remove this customTheme, I was playing with colors.

File: graphql-dxm-provider/src/main/resources/tools/graphql-playground.jsp [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            };
+
+            window.addEventListener('load', function() {
+                GraphQLPlayground.init(document.getElementById('graphql-playground'), {
+                    endpoint: '${graphQlEndpoint}',
+                    settings: {
+                        'request.credentials': 'same-origin'
+                    },
+                    shareEnabled: true,
+                    //workspaceName: 'Jahia',
```
Comment (cedricmoitrier) [inline graphql-dxm-provider/src/main/resources/tools/graphql-playground.jsp:51]: You can remove both commented attributes (workspaceName and codeTheme)

File: graphql-dxm-provider/src/main/resources/tools/graphql-playground.jsp [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                // rightDrawerBackground: ''
+            };
+
+            window.addEventListener('load', function() {
+                GraphQLPlayground.init(document.getElementById('graphql-playground'), {
+                    endpoint: '${graphQlEndpoint}',
+                    settings: {
+                        'request.credentials': 'same-origin'
+                    },
+                    shareEnabled: true,
```
Comment (cedricmoitrier) [inline graphql-dxm-provider/src/main/resources/tools/graphql-playground.jsp:50]: This one enables integration with https://www.graphqlbin.com/ to share a workspace. Don't know whether or not we should enable that. Should double check with @fpral

**Review by cedricmoitrier — COMMENTED** [review]
I don't recall exactly why I did not use 1.8.x branch in the end (stability?). But a 1.7.26 has been released in between, thus we should use this one over the 1.7.23 one.

File: graphql-dxm-provider/src/main/resources/tools/graphql-playground.jsp (hunk 3 above)
Comment (Yousria) [reply graphql-dxm-provider/src/main/resources/tools/graphql-playground.jsp:50]: > I don't recall exactly why I did not use 1.8.x branch in the end (stability?). But a 1.7.26 has been released in between, thus we should use this one over the 1.7.23 one.

I couldn't use 1.7.26 version tried to make it work but It doesn't load constantly, here is a screenshot
Capture d’écran 2020-09-01 à 09 07 55

**Review by cedricmoitrier — APPROVED** [review]

**Pull request merged**